src/main/java/org/folio/services/finance/transaction/PendingPaymentWorkflowService.java

```diff
@@ -126,23 +79,30 @@ private Future<Void> cleanupOldEncumbrances(List<InvoiceWorkflowDataHolder> hold
       .collect(toList());
     String fiscalYearId = invoice.getFiscalYearId();
     return encumbranceService.getEncumbrancesByPoLineIds(poLineIds, fiscalYearId, requestContext)
-      .compose(transactions -> cleanupOldEncumbrances(transactions, holders, requestContext));
+      .compose(transactions -> cleanupOldEncumbrances(transactions, holders, requestContext))
+      .recover(t -> {
+        logger.error("cleanupOldEncumbrances: Failed to release encumbrances", t);
```

Comment:
since we have poLinIds here, lets add them into error log

Reply:
OK